client/src/Components/settings/forms/AddCompanyForm.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from "react";
+import { TextField, Grid } from "@material-ui/core";
+import FormButton from "./FormButton";
+
+const AddCompanyForm = ({ classes, addCompany }) => {
+  const [input, setInput] = useState("");
+  return (
+    <Grid className={classes.inputWrapper}>
+      <TextField
+        variant="outlined"
+        className={classes.formField}
+        label="Company Name"
+        value={input}
+        onChange={(e) => setInput(e.target.value)}
+        InputLabelProps={{
+          style: {
+            fontWeight: "bold",
+          },
+        }}
```

Comment:
I have a custom textfield component in the components folder that I used for the login/signup pages. Is that usable here?

Reply:
It should be. Don't know why I didn't think of that sooner. I'm going to try it out and if no probs will be using your component

Reply:
I think when I started out with the ticket I didn't know how to proceed so I might've done some base components from scratch

Reply:
Actually, to get the input text to fit the spec I used some extra props asked by `TextField`(InputLabelProps & InputProps), so maybe not yet :/

Reply:
I see. No worries! Just curious. I think there should be a way to modify the component I have to fit needs, but if it's easier to write a new one that makes sense.